packages/muelu/doc/UsersGuide/masterList.xml

```diff
@@ -892,6 +892,14 @@
     </parameter>
 
     <parameter>
+      <name>repartition: target rows per proc</name>
+      <type>int</type>
+      <default>24000</default>
```

Comment:
This is a significant departure from the current behavior.  To preserve backwards compatibility, shouldn't the default be 1000, or whatever the default for min rows per proc is?